third_party/proto/ibc/applications/transfer/v1/denomtrace.proto

```
@@ -0,0 +1,16 @@
+syntax = "proto3";
+
+package ibc.applications.transfer.v1;
```

Buf (1.47.2)

Files with package "ibc.applications.transfer.v1" must be within a directory "ibc/applications/transfer/v1" relative to root but were in directory "third_party/proto/ibc/applications/transfer/v1".

(PACKAGE_DIRECTORY_MATCH)